portfolio/src/main/webapp/script.js

```diff
@@ -11,84 +11,165 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
+/* 
+ * Get the login status of the current user. If logged in, display the comment
+ * submission input box and the button to select number of displayed comments.
+ * Display a logout button. If not logged in, hide the comment submission and 
+ * selection of number of displayed comments. Display a login button.
+ */
+function getLoginStatus() {
+    fetch('/login')
+    .then(response => response.json())
+    .then((result) => {
+        const button_area = document.getElementById('button-area');
+        const msg_form = document.getElementById('msg-form');
+        const comment_form = document.getElementById('comment-form');
+        const quantity_input = document.getElementById('quantity');
+        let button = document.createElement('button');
+        button_area.innerHTML = '';
+ 
+        if (result.loggedIn == 1) {
+            /* Allow logged in users to submit comments and change number of comments. */
+            comment_form.style.display = 'flex';
+            quantity_input.style.display = 'block';
```

Comment:
How about avoiding this by having separate div elements for the login and logout?

Reply:
I wanted to avoid doing this because there's a lot of repetitive code. The only changes between login/logout would be changing a name and a URL, for instance, so I figured it would be better to just change those vs. having redundant code. Please let me know if there are more advantages to using two divs that I'm unaware of, though!

Reply:
Although this might be a good idea specifically for the login and logout buttons themselves. I was struggling on how to set specific attributes for these and that's why I ended up keeping them in the JavaScript file, but I'll try using two divs and see if that's easier.